libs/platform/src/lib/components/table/components/table-toolbar/table-toolbar.component.html

```diff
@@ -23,6 +23,7 @@
             placeholder="Search"
             [contentDensity]="_getCozyCompactSize(size)"
             [suggestions]="searchSuggestions"
+            [ariaLabel]="_getSearchAriaLabel(counter)"
```

Comment:
Let's generate unique id for the current toolbar label and pass this id to the `fdp-search-field` as `aria-labelledby` option.
In this case we don't need `_getSearchAriaLabel()`